.gitignore

```diff
@@ -377,3 +377,6 @@ mail.md
 
 *.db-shm
 *.db-wal
+
+
+/src/AdminConsole/wwwroot/css/tailwind.css
```

Comment:
Maybe makes sense to have a local `.gitignore` file inside AdminConsole for this?